dev/bin/pyenv.reset.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Is this file for local use or needed in the docker image? In the former case perhaps it shouldn't be committed to genienlp.